src/llms/openai.ts

```diff
@@ -226,19 +226,19 @@ export interface ChatCompletionsChunk {
 }
 
 /** Return true if the message is a 'content' message. */
-export function isContentMessage(message: any): message is ContentMessage {
-  return message.content != null;
+export function isContentMessage(message: ChatCompletionsDelta): message is ContentMessage {
+  return 'content' in message
 }
 
 
 /** Return true if the message is a 'done' message. */
-export function isDoneMessage(message: any): message is DoneMessage {
-  return message.done !== undefined
+export function isDoneMessage(message: ChatCompletionsDelta): message is DoneMessage {
+  return 'done' in message;
```

Comment:
Hmm wouldn't this still return `true` if `message.done = null`? Should we keep an undefined/null check?

Reply:
Yeah I guess it would, but OpenAI's API never returns `"done": null` afaik 🤷. Could keep the null check though yeah